rest/doc_api.go

```diff
@@ -470,7 +470,7 @@ func (h *handler) handleDeleteDoc() error {
 	}
 	newRev, err := h.db.DeleteDoc(docid, revid)
 	if err == nil {
-		h.writeJSON(db.Body{"ok": true, "id": docid, "rev": newRev})
+		h.writeBytesAsJSONResponse(http.StatusOK, []byte(`{"ok": true, "id":"`+docid+`", "rev":"`+newRev+`"}`))
```

Comment:
Let's keep the alphabetical ordering of properties that stdlib marshalling would produce, as well as trimming any whitespace:

```suggestion
		h.writeBytesAsJSONResponse(http.StatusOK, []byte(`{"id":"`+docid+`","ok":true,"rev":"`+newRev+`"}`))
```

Applies to the other usages too.

Reply:
Ah yeah, thats my bad. Just copied how it was in the previous body.